source/braille.py

```diff
@@ -1713,6 +1733,7 @@ def _updateDisplay(self):
 			wx.CallAfter(self._cursorBlinkTimer.Start,blinkRate)
 
 	def _writeCells(self, cells):
+		brailleViewer.update(cells, self._rawText)

```

Comment:
I"m worried that this could somehow slow down braille output, as braille output is blocked by the braille viewer being updated. Have you considered rate limiting here?

Reply:
I'd like to see that there is a problem to solve first. Bear in mind, most people wont be using the braille viewer and a display. If testing shows there is a problem, then we have a benchmark that we can try to improve on.